kotlin/query-with-cql-filters/src/main/java/com/esri/arcgisruntime/sample/querywithcqlfilters/MainActivity.kt

```diff
@@ -0,0 +1,374 @@
+/* Copyright 2017 Esri
```

Comment:
this is a new sample, so the copyright should reflect the year the sample was created. In samples where we are updating sample code, we leave the copyright year as is. 

```suggestion
/* Copyright 2021 Esri
```